content/applications/finance/expenses/post_expenses.rst

```diff
@@ -0,0 +1,74 @@
+=============
+Post expenses
+=============
+
+Once an expense report is :ref:`approved <expenses/approve>`, the next step is to post the expense
+report to the proper accounting journal.
+
+.. important::
+   To post expense reports to an accounting journal, the user **must** have the following
+   :doc:`access rights <../../general/users/access_rights>`:
+
+   - Accounting: Accountant or Adviser
+   - Expenses: Manager
+
+Only expense reports with an :guilabel:`Approved` status can post the expenses to a journal. To view
+all expense reports, navigate to :menuselection:`Expenses app --> Expense Reports`. Next, to view
+**only** approved expense reports that need to be posted, adjust the filters on the left side so
+that only the :guilabel:`Approved` checkbox is ticked.
+
+.. image:: post-expenses/post-reports.png
+   :align: center
+   :alt: View reports to post by clicking on expense reports, then reports to post.
+
+.. note::
+   The default :guilabel:`All Reports` dashboard displays all expense reports, except reports with a
+   status of :guilabel:`Refused`.
+
+Expense reports can be posted to accounting journals in two ways: :ref:`individually
+<expenses/individual>` or :ref:`in bulk <expenses/multiple>`.
+
+.. _expenses/individual:
+
+Post individual reports
+-----------------------
+
+To post an individual report, navigate to :menuselection:`Expenses app --> Expense Reports`, and
+click on an individual report to view the report form. In this view, several options are presented:
+:guilabel:`Post Journal Entries`, :guilabel:`Report In Next Payslip`, :guilabel:`Refuse`, or
+:guilabel:`Reset to Draft`.
```

Comment:
Wouldn't the chosen report (from the All Reports page) **need** to be 'Approved' in order for all these options/buttons to appear at the top of the form? Worth looking into during your revisions. 👍